dockergen/nginx.tmpl

```diff
@@ -71,6 +72,7 @@ server {
 {{ range $url, $containers := groupByMulti $containersByHost "Env.VIRTUAL_URL" "," }}
     location {{ $url }} {
         proxy_pass http://{{ $host }}{{ replace $url "/" "-" -1 }}u/;
+        proxy_redirect https://{{ $host }}{{ replace $url "/" "-" -1 }}u/ /;
```

Comment:
I will investigate it in the upcoming days. The container that I had problems with is a GitLab container that I do not really know well so I have no idea if the issue is general or container specific.